blockstack-sdk/src/main/java/org/blockstack/android/sdk/model/PutFileOptions.kt

```diff
@@ -7,8 +7,9 @@ import org.json.JSONObject
  *
  * @property encrypt encrypt the with the private key of the current user before writing to storage
  * @property contentType contentType of file to be used only if not encrypted
+ * @property sign sign the data using ECDSA on SHA256 hashes with the user's app private key. If a string is specified, it is used as the private key.
```

Comment:
The current documentation for this property is unclear. "If a string is specified" what does this mean? Are you saying this property can take `true` literal string or an arbitrary string?

If `sign` is `true`and the `getFileOptions` `verify` property is *also* `true`, then the signature is verified -- is this correct?

We need to expand the description to reference the interaction with getFileOptions

Reply:
@moxiegirl Yes, that is correct.

What about:

"if set to true the data is signed using ECDSA on SHA256 hashes with the user's app private key. If a string is specified, it is used as the private key instead of the user's app private key. The signature can be verified with `GetFileOptions.verify` set to true when retrieving the data"

The documentation was taken from blockstack.js. Probably that could get an update as well.

Reply:
LGTM